app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +28,7 @@
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
+                <category android:name="android.intent.category.LEANBACK_LAUNCHER"/>
```

Comment:
Have you checked it on Android TV emulator, because for me it's just a blank screen on TV emulator.

Reply:
I ran it and it works fine :)